apps/heft/src/plugins/SassTypingsPlugin/SassTypingsPlugin.ts

```diff
@@ -42,10 +42,18 @@ export class SassTypingsPlugin implements IHeftPlugin {
       buildFolder: heftConfiguration.buildFolder,
       sassConfiguration
     });
-    await sassTypingsGenerator.generateTypingsAsync();
-    if (isWatchMode) {
-      await sassTypingsGenerator.runWatcherAsync();
-    }
+    await new Promise((resolve: () => void, reject: (error: Error) => void) => {
+      sassTypingsGenerator
+        .generateTypingsAsync()
+        .then(() => {
+          resolve();
+          if (isWatchMode) {
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
This lint rule is telling you that if `runWatcherAsync()` rejects its promise (e.g. because of an uncaught exception) then the entire Heft process will crash, because there's no catch handler.

Think twice before ignoring this important lint rule. :-)